code/Rmd/CountMatrixmiRNA.Rmd

```diff
@@ -0,0 +1,127 @@
+---
+title: "miRNA descriptive analysis"
+author: "Lorena Pantano"
+date: "02/09/2015"
+output: html_document
+---
+
+```{r libs, warning=FALSE, message=FALSE}
+library(tidyr)
+library(dplyr)
+library(knitr)
+library(synapseClient)
+library(DESeq2)
+library(vcd)
+# library(CHBUtils)
+library(corrplot)
+
+synapseLogin()
+
+options(xtable.type="html")
+
+knitr::opts_chunk$set(
+  echo=TRUE,
+  warning=FALSE,
+  message=FALSE,
+  error = FALSE,
+  tidy = FALSE)
+```
+
+# Load Data
+
+## Get metadata and count matrix 
+
+```{r get-id}
+Query <- "select id,UID,name,fileType,dataType from file where name=='Summarized miRNA Counts' AND  projectId=='syn1773109'"
```

Comment:
Not really a safe way to get it, will break if there happen to be files called the same! Better to use the syn ID directly, it won't change; or at least further limit by parentId